docs/src/tutorials/linear-model.rst

```diff
@@ -0,0 +1,4 @@
+Equivariant Linear Model
+------------------------
+
+TBD
```

Comment:
Since I am currently doing a Ridge implementation in equisolve, I am asking myself if an example really belongs here or do we just link to equisolve on this page?

Reply:
My idea was more to provide a simple example using `equistore.operations` directly. Then the doc should 100% link to equisolve as a more complete implementation of this!

Reply:
Okay make sense.